sdk/src/main/java/io/customer/sdk/data/request/DeliveryEvent.kt

```diff
@@ -0,0 +1,23 @@
+package io.customer.sdk.data.request
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import java.util.*
+
+@JsonClass(generateAdapter = false)
+internal enum class DeliveryType {
+    in_app
+}
+
+@JsonClass(generateAdapter = true)
+internal data class DeliveryPayload(
+    @field:Json(name = "delivery_id") val deliveryID: String,
+    val event: MetricEvent,
```

Comment:
For iOS, I had to create a new `Metric` object specifically for in-app because in-app has `clicked` and push doesn't.

Reply:
Same as the previous comment.